avocado/core/job.py

```diff
@@ -548,8 +560,7 @@ def run(self):
             self.exitcode |= exit_codes.AVOCADO_FAIL
             return self.exitcode
         finally:
-            if self.job_pre_post_dispatcher is not None:
-                self.job_pre_post_dispatcher.map_methods('post', self)
+            self.post_tests()
```

Comment:
There you go. I know the patches are chronological, but for me it'd look better to merge this one to the `pre` one as `pre` is nothing without `post`.